frontend/annotation_resolver.py

```diff
@@ -65,27 +69,27 @@ def resolve(self, annotation):
         if annotation[:4] == "List":
```

Comment:
White spaces will never matter, because the type annotation are given as AST nodes which are unparsed to text.

So List[int] will be given as the following node:
```python
Subsciprt(
    value=Name(
                  id="List"
              ),
    slice=Index(
                  value=Name(
                                 id="int"
              )
)
```
And AST constructs ignore any white spaces.
As a result `List[ int]`, `List[    int  ]`, `List[int, ]` are all accepted.